docs/UserGuide.md

```diff
@@ -267,4 +282,5 @@ Action                | Format, Examples
 **List all orders**             | `listOrder INDEX`<br> e.g., `listOrder 3`
 **Delete an order**             | `deleteOrder ORDER_INDEX`<br> e.g., `deleteOrder 2`
 **Mark an order as completed**  | `markOrder ORDER_INDEX`<br> e.g., `markOrder 2`
+**Unmark an order as completed**| `unmarkOrder ORDER_INDEX`<br> e.g., `unmarkOrder 2`
```

Comment:
As per unmarkCommand messages, unmark should be understood as removing the `Completed` status/mark